webapp/src/components/Container.tsx

```diff
@@ -351,7 +351,7 @@ const Container: React.FC<ContainerProps> = ( { children } ) => {
               </ListItemButton>
             </ListItem>
           </List>
-          { screenState.isAuthed ?
+          { screenState.isAuthed && isEducator(screenState.roles) ?
```

Comment:
we need to wrap this in a useCallback to maintain state

Reply:
> we need to wrap this in a useCallback to maintain state

done